pkgs/top-level/unix-tools.nix

```diff
@@ -49,12 +49,12 @@ in rec {
     linux = pkgs.utillinux;
   };
   getconf = singleBinary "getconf" {
-    linux = if hostPlatform.libc == "glibc" then lib.getBin pkgs.glibc
+    linux = if stdenv.cc.libc == "glibc" then lib.getBin pkgs.glibc
```

Comment:
`stdenv.cc.libc` is a derivation :D, you'll want to go back to how it was before.